### PR TITLE
Seed phrase verified

### PR DESCRIPTION
**Pull request by adrianbona**

### CODE REVIEW

**Review by ACatThatPrograms — CHANGES_REQUESTED** [review]
Just the switch to util function for ```className()``` as opposed to a whole extra package

File: app/client/src/pages/SeedPhraseVerified.jsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                </Grid.Column>
+
+            </Grid>
+
+        </Page>
+    )
+
+}
+
+export default SeedPhraseVerified;
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply app/client/src/pages/SeedPhraseVerified.jsx:140]: Updated and removed some warning from the console as well.

File: app/client/package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "homepage": "./",
   "dependencies": {
     "big-integer": "^1.6.48",
+    "classnames": "^2.3.1",
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply app/client/package.json]: Updated and removed some warning from the console as well.

**Review by ACatThatPrograms — CHANGES_REQUESTED** [review]
Need them utility files :joy_cat:

**Comment by ACatThatPrograms** [comment]
:100:  Niceeee! Enjoy your weekend.

**Pull request merged**